app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -246,7 +273,11 @@ class OverdueScreen : BaseScreen<
   }
 
   override fun showOverdueAppointments(overdueAppointmentSections: OverdueAppointmentSections) {
-    // TODO: Bind UI
+    overdueRecyclerView.visibility = View.VISIBLE
```

Comment:
Just commenting this here so that we don't forget. We have to handle the hiding of the recycler view as well.

Reply:
Yep, I am hoping we can do it #3913 where we handle the no overdue state? When there are overdue patients we go with the usual flow. If there aren't, we hide the list and show the illustration. WDYT?